src/pages/view-quotes/view-quotes.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { BlockQuote } from './block-quote';
+import { fetchQuotes } from '../../api/fetch-quotes';
+import './styles.css';
+
+class ViewQuotes extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      quotes: [],
+      quote: {
+        text: 'Java is to JavaScript what Car is to Carpet.',
+        author: 'A JS Developer',
+      },
+      displayQuoteBox: true,
+    };
+    this.randomQuote = this.randomQuote.bind(this);
+  }
+
+  componentDidMount() {
+    this.onGetQuotes();
+  }
+
+  async onGetQuotes() {
+    const quotes = await fetchQuotes();
+    this.setState({ quotes }, this.randomQuote);
+  }
+
+  setQuote(quote) {
+    this.setState({ quote, displayQuoteBox: true });
+  }
+
+  randomQuote() {
```

Comment:
Use shortHand Class Syntax and arrow-functions for binding methods to the class

chooseRandomQuote = () => {
}